galette/templates/default/elements/header.html.twig

```diff
@@ -14,7 +14,6 @@
     {% if file_exists(localstylesheet) %}
         <link rel="stylesheet" type="text/css" href="{{ base_path() }}/{{ constant('GALETTE_THEME') }}galette_local.css" />
     {% endif %}
-        <link rel="stylesheet" type="text/css" href="{{ base_path() }}/assets/css/galette_print.css" media="print" />
```

Comment:
Not sure I understand this removal; is that replaced another way in the framework?

Reply:
I moved it in a media query in the theme : https://github.com/galette/galette/pull/229/commits/9789fd8ef9607f8f38fd6ec42d85ea4c771f9b60#diff-e3ee00f5809b9740fa5b5966c7c1f2fa0a7c7ece5a007b63b06e2c8bf79f88f8 :wink: